client/src/app/[locale]/(admin)/dashboard/config.ts

```diff
@@ -0,0 +1,90 @@
+import * as Yup from 'yup';
+import { TranslationValues } from 'next-intl';
+import { isValidPhoneNumber } from 'libphonenumber-js';
+
+interface FormValues {
+  name: string;
+  site: string;
+  email: string;
+  phone: string;
+  agree: boolean;
+  lastName: string;
+  middleName: string;
+  socialNetworks: string;
+  organizationName: string;
+  positionOrganization: string;
+  organizationTaxNumber: string;
+  certificateOfRegister: string;
+  dateOfRegisterOrganization: string;
+}
+
+export const initialValues = (data?: FormValues) => ({
+  name: data?.name ?? '',
+  site: data?.site ?? '',
+  email: data?.email ?? '',
+  phone: data?.phone ?? '',
+  agree: data?.agree ?? '',
+  lastName: data?.lastName ?? '',
+  middleName: data?.middleName ?? '',
+  socialNetworks: data?.socialNetworks ?? '',
+  organizationName: data?.organizationName ?? '',
+  positionOrganization: data?.positionOrganization ?? '',
+  organizationTaxNumber: data?.organizationTaxNumber ?? '',
+  certificateOfRegister: data?.certificateOfRegister ?? '',
+  dateOfRegisterOrganization: data?.dateOfRegisterOrganization ?? '',
+});
+
+export const validationSchema = (
```

Comment:
The name of the constant doesn't provide an information to whom those validation belong. It'd be more beneficial to rename it.